packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -251,6 +253,10 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
             connectMessage.supportedFeatures[feature_get_ops] = true;
         }
 
+        if (mc.config.getBoolean("Fluid.Driver.Odsp.LightOpAcks") !== false) {
```

Comment:
I'd suggest not using feature gate here, i.e. shift it to service. I think this code should just tell service that client is capable of working with such content, and service should make a call if it wants to leverage that.

Reply:
Is the best way to do that through the `socket.emit(...)` call? My thinking was that service might not know to interpret this, so I added the flag to wait until it can

Reply:
I think the best flow is the following:
1. Client advertise (on connection - "connect_document" / IConnect) that it supports certain feature 
2. Ordering service instructs client ("connect_document_success" / IConnected)
3. Client changes behavior based on # 2.

Here, I believe, the only client change in behavior is tracking more state in memory, i.e. there is really no protocol changes in terms of client outbound traffic. Only inbound traffic shape is changed - service does it in response to # 1.